app/Http/Controllers/AdminDash.php

```diff
@@ -635,7 +635,7 @@ public function storeVisitor(Request $request) {
         
         // Add to the VATUSA roster
         $client = new Client();
-        $res = $client->request('POST', 'https://api.vatusa.net/v2/facility/'.Config::get('vatusa.facility').'/roster/manageVisitor/'.$request->input('cid').'?apikey='.Config::get('vatusa.api_key'), ['http_errors' => false]);
+        $res = $client->request('POST', Config::get('vatusa.base').'/v2/facility/'.Config::get('vatusa.facility').'/roster/manageVisitor/'.$request->input('cid').'?apikey='.Config::get('vatusa.api_key'), ['http_errors' => false]);
```

Comment:
See comment above - move to separate issue.

Reply:
Removed